addons/mail/tests/test_mail_message_format.py

```diff
@@ -0,0 +1,33 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import Command
+from odoo.tests.common import HttpCase, tagged, new_test_user
+
+
+@tagged("post_install")
+class TestMailMessageFormat(HttpCase):
+    def test_add_livechat_username_to_author(self):
+        operator = new_test_user(self.env, login=f"Marc", livechat_username="marco")
+        operator.partner_id = self.env["res.partner"].create({"name": "marc"})
+        livechat = self.env["im_livechat.channel"].create({"name": "Livechat", "user_ids": [operator.id]})
+        self.env["bus.presence"].create({"user_id": operator.id, "status": "online"})
+        channel_info = self.make_jsonrpc_request(
```

Comment:
Can't you write the test to be simpler and skip the jsonrpc ? Seems it complicates the test without much value as in the end you only need a channel_id, unless there are hidden things ?